config.yaml

```diff
@@ -58,7 +58,13 @@
 # Defaults to: none
 # Overwrite with environment variable: DEVICEAUTH_TENANTADM_ADDR
 
-# device_auth_orchestrator:  http://tenantadm
+# tenantadm_addr:  http://tenantadm
+
+# Default tenant token, for devices that don't supply one (optional)
+# Defaults to: none
+# Overwrite with environment variable: DEVICEAUTH_DEFAULT_TENANT_TOKEN
+
+# default_tenant_token:  abcdef
```

Comment:
I'm not sure about this comment
I'm afraid it may suggest that the default_tenant_token may be anything, but the default_tenant_token needs to be a valid tenant token (valid jwt token with tenant claim)

Reply:
Good point, I will replace it with:

```
# default_tenant_token:  <VALID_TENANT_TOKEN>
```